pkg/cache/v3/linear.go

```diff
@@ -256,7 +282,16 @@ func (cache *LinearCache) computeResourceChange(sub Subscription, useResourceVer
 			}
 		}
 	} else {
-		for resourceName := range sub.SubscribedResources() {
+		combinedResources := maps.Clone(sub.SubscribedResources())
```

Comment:
`combinedResources` is not really descriptive in the context of this method. Maybe `allConsiderableResources` or something like this